### ISSUE TITLE
ImportError

### ISSUE THREAD

**Issue by ElinorG11** [IBM/aihwkit#298]
Hello,

I'm using Linux Ubuntu 20.04 with GPU Nvidia RTX 3070 and cuda version 11.4. PyTorch was installed as follows:

I've installed the `aihwkit` as was specified in the Development setup guide. After successful installation (also installed all the requirements as specified in the git), I didn't change any of the code, just tried to run some of the examples. I get the following error:

```
Traceback (most recent call last):
  File "./01_simple_layer.py", line 25, in <module>
    from aihwkit.nn import AnalogLinear
  File "/home/elinor/aihwkit/src/aihwkit/nn/__init__.py", line 17, in <module>
    from aihwkit.nn.modules.container import AnalogSequential
  File "/home/elinor/aihwkit/src/aihwkit/nn/modules/container.py", line 23, in <module>
    from aihwkit.nn.modules.base import AnalogModuleBase
  File "/home/elinor/aihwkit/src/aihwkit/nn/modules/base.py", line 21, in <module>
    from aihwkit.simulator.configs import (
  File "/home/elinor/aihwkit/src/aihwkit/simulator/configs/__init__.py", line 15, in <module>
    from .configs import (
  File "/home/elinor/aihwkit/src/aihwkit/simulator/configs/configs.py", line 18, in <module>
    from aihwkit.simulator.configs.devices import (
  File "/home/elinor/aihwkit/src/aihwkit/simulator/configs/devices.py", line 25, in <module>
    from aihwkit.simulator.configs.helpers import (
  File "/home/elinor/aihwkit/src/aihwkit/simulator/configs/helpers.py", line 20, in <module>
    from aihwkit.simulator.rpu_base import devices, tiles

ImportError: /home/elinor/aihwkit/src/aihwkit/simulator/rpu_base.cpython-38-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor6deviceEv
```

I'm not sure what can be the source of the problem, and there's no search results in google regarding it. perhaps you have any idea what could it be?

could it be a compilation problem? I compiled with the following command:
`python setup.py build_ext --inplace -DUSE_CUDA=ON`

with `aihwkit_env` activated, and the pwd was `~/aihwkit` which contains the folder `src` where the cloned source code is. should I compile it from another location?

_________________________________________________________________________________________________
UPDATE SOLUTION:
It was a compilation problem. I found out that I have few python interpreter versions on my computer (2.7, 3.8) and `aihwkit` requires >=3.7 (as specified in the `setup.py` code). I recompiled using the following command (notice I used `python3` instead of `python`):

`(aihwkit_env) ~/aihwkit$ python3 setup.py build_ext --inplace -DUSE_CUDA=ON`

 after I've ensured I'm running in the adequate virtual environment `aihwkit_env` and the location from which I run the command was `/home/username/aihwkit` where the `src` directory with the cloned source code is stored.
__________________________________________________________________________________________________

Thank you

**Comment by ElinorG11** [comment]
SOLUTION:
It was a compilation problem. I found out that I have few python interpreter versions on my computer (2.7, 3.8) and `aihwkit` requires >=3.7 (as specified in the `setup.py` code). I recompiled using the following command (notice I used `python3` instead of `python`):

`(aihwkit_env) ~/aihwkit$ python3 setup.py build_ext --inplace -DUSE_CUDA=ON`

 after I've ensured I'm running in the adequate virtual environment `aihwkit_env` and the location from which I run the command was `/home/username/aihwkit` where the `src` directory with the cloned source code is stored.
